src/mame/heathkit/tlb.h

```diff
@@ -209,14 +215,14 @@ class heath_watz_tlb_device : public heath_tlb_device
 class heath_ultra_tlb_device : public heath_tlb_device
 {
 public:
-	heath_ultra_tlb_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock = 0);
+	heath_ultra_tlb_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock = 0);
 
 protected:
 	virtual const tiny_rom_entry *device_rom_region() const override;
 	virtual ioport_constructor device_input_ports() const override;
 	virtual void device_add_mconfig(machine_config &config) override;
 
-	void mem_map(address_map &map);
+	void         mem_map(address_map &map);
```

Comment:
See here’s an example where you’ve arbitrarily lined up `device_add_mconfig` and `mem_map`, but you haven’t lined up `device_rom_region` and `device_input_ports` with them.  It’s a bit concerning if even the initial tabulation is inconsistent.